docs/tools/replace.sh

```diff
@@ -70,21 +101,74 @@ function init() {
 }
 
 function main() {
-    node="$1"
-    src_drive="${2#/dev/}"
-    dest_drive="${3#/dev/}"
-
-    # Get source drive ID
-    src_drive_id=$(get_drive_id "${node}" "${src_drive}")
-    if [ -z "${src_drive_id}" ]; then
-        echo "source drive ${src_drive} on node ${node} not found"
+    src_drive="${1#/dev/}"
+    dest_drive="${2#/dev/}"
+    node="${3}"
+
+    if [ "${src_drive}" == "${dest_drive}" ]; then
+        echo "the source and destination drives are same"
+        exit 255
+    fi
+
+    if ! is_uuid "${src_drive}"; then
+        if [ -z "${node}" ]; then
+            echo "node argument must be provided for drive name"
+            exit 255
+        fi
+        # Get source drive ID
+        mapfile -t src_drive_ids < <(get_drive_ids "${node}" "${src_drive}")
```

Comment:
1. use `src_drive_ids=( $(get_drive_ids "${node}" "${src_drive}") )` and ignore ` # shellcheck disable=SC2207`. This way you could use `dest_drive_id="${dest_drive_ids[0]}"`
2. This code block repeats for destination drive as well. You could have a common function.